planner/core/exhaust_physical_plans.go

```diff
@@ -223,9 +233,33 @@ func getNewJoinKeysByOffsets(oldJoinKeys []*expression.Column, offsets []int) []
 	return newKeys
 }
 
+func getNewNullEQByOffsets(oldNullEQ []bool, offsets []int) []bool {
+	newNullEQ := make([]bool, 0, len(oldNullEQ))
+	for _, offset := range offsets {
+		newNullEQ = append(newNullEQ, oldNullEQ[offset])
+	}
+	for pos, key := range oldNullEQ {
+		isExist := false
+		for _, p := range offsets {
```

Comment:
Does `oldNullEQ` and `offsets` have same order regarding the offset? if yes, we can just loop over `offsets` and check if corresponding item exists in `oldNullEQ` by slice offset?